cloud/ovirt/ovirt_vms.py

```diff
@@ -0,0 +1,583 @@
+            - "Number of virtual CPUs sockets of the Virtual Machine. Default value is set by oVirt engine."
+    type:
+        description:
+            - "Type of the Virtual Machine. Default value is set by oVirt engine."
+        choices: [server, desktop]
+    operating_system:
+        description:
+            - "Operating system of the Virtual Machine. Default value is set by oVirt engine."
+    host:
+        description:
+            - "Specify host where Virtual Machine should be running. By default the host is chosen by engine scheduler."
+            - "This parameter is used only when C(state) is I(running) or I(present)."
+    paused:
+        description:
+            - "If I(True) Virtual Machine will be started in paused mode."
+            - "This parameter is used only when C(state) is I(running), I(present) or I(stopped) and VM didn't exist before."
+        default: False
+    high_availability:
+        description:
+            - "If I(True) Virtual Machine will be set as highly available."
+            - "If I(False) Virtual Machine will be unset as highly available."
+            - "If no value is passed, default value is set by oVirt engine."
+    stateless:
+        description:
+            - "If I(True) Virtual Machine will be set as stateless."
+            - "If I(False) Virtual Machine will be unset as stateless."
+            - "If no value is passed, default value is set by oVirt engine."
+    clone:
+        description:
+            - "If I(True) then the disks of the created virtual machine will be cloned and independent of the template."
+            - "This parameter is used only when C(state) is I(running) or I(present)."
+        default: False
+    clone_permissions:
+        description:
+            - "If I(True) then the permissions of the template (only the direct ones, not the inherited ones)
+            will be copied to the created virtual machine."
+            - "This parameter is used only when C(state) is I(running) or I(present)."
+        default: False
+    force:
+        description:
+            - "If I(True) Virtual Machine will be rebooted after its parameters are updated if VM is I(running)."
+        default: False
+    nics:
+        description:
+            - "List of NICs, which should be attached to Virtual Machine. NIC is described by following dictionary:"
+            - "C(name) - Name of the disk"
+            - "C(profile_name) - Profile name where NIC should be attached"
+            - "C(interface) -  Type of the network interface. One of following: I(virtio), I(e1000), I(rtl8139), default is I(virtio)"
+            - "C(mac_address) - Custom MAC address of the network interface, by default it's obtained from MAC pool"
+            - "C(Note:)"
+            - "This parameter is used only when C(state) is I(running) or I(present) and is able to only attach NICs.
+               To manage NICs of the VM in more depth please use M(ovirt_nics) module instead."
+    disks:
```

Comment:
Is this option limited to just pre-created disks? Being able to define a very simple disk here would be helpful for simple use cases.